spec/README.mediawiki

```diff
@@ -1130,30 +1152,99 @@ DEX will send a list of '''Match objects'''.
 |-
 | rate      || int    || the rate that the order matched at (as quote asset per unit base asset, atoms)
 |-
-| address   || string || the client's receiving address
+| address   || string || the counterparty's receiving address
+|-
+| sig       || string || DEX's hex-encoded signature of the serialized notification data. serialization described below
+|}
+
+'''Match notification serialization'''
+
+{|
+! field      !! size (bytes)  !! description
 |-
-| timestamp || int    || UTC timestamp. order broadcast time
+| orderid    || 32 || the order ID
+|-
+| matchid    || 8  || the ID assigned to this match
 |-
-| sig       || string || server hex-encoded signature of the serialized notification data. serialization described below
+| quantity   || 8  || the matched amount, in atoms of the base asset
+|-
+| rate       || 8  || the rate that the order matched at
+|-
+| timestamp  || 8  || the client's UNIX timestamp
```

Comment:
For the server to verify the signature in the clients acknowledgement of the match notification, it needs to have the timestamp.  Is the timestamp the time that the match was made or the time that the client received the match and generated the ack?

Reply:
Good call. Should be the server's timestamp.